pallets/collator-selection/src/mock.rs

```diff
@@ -177,12 +178,25 @@ impl pallet_session::Config for Test {
 	type WeightInfo = ();
 }
 
+type CandidateBagsListInstance = pallet_bags_list::Instance1;
+impl pallet_bags_list::Config<CandidateBagsListInstance> for Test {
+	type RuntimeEvent = RuntimeEvent;
+	/// The voter bags-list is loosely kept up to date, and the real source of truth for the score
+	/// of each node is the staking pallet.
```

Comment:
What do you mean? We don't have the staking pallet.

Reply:
It's meant to be
```
/// The candidate bags-list is loosely kept up to date, and the real source of truth for the score
/// of each node is the collator-selection pallet.
```
It's a copy-paste error, I'll fix it.